config/cmake/modules/MfemCmakeUtilities.cmake

```diff
@@ -783,6 +783,19 @@ function(mfem_get_target_options Target CompileOptsVar LinkOptsVar)
     # INTERFACE_LINK_DEPENDS
     # INTERFACE_LINK_DIRECTORIES
     # INTERFACE_LINK_OPTIONS
+  elseif("${type}" STREQUAL "UNKNOWN_LIBRARY")
+    get_target_property(Location ${tgt} LOCATION)
+    if (Location)
+      get_filename_component(Dir ${Location} DIRECTORY)
+      get_filename_component(NameWE ${Location} NAME_WE)
+      string(REGEX REPLACE "^lib" "" LibName ${NameWE})
+      list(APPEND LinkOpts
+        "-L\"${Dir}\""
+        "${shared_link_flag}\"${Dir}\""
+        "-l${LibName}")
```

Comment:
My initial reaction is that changing the conditional statement would be better than copy-paste of the conditional body, i.e., changing
```
elseif ("${type}" STREQUAL "SHARED_LIBRARY")
```
to something like
```
elseif ("${type}" STREQUAL "SHARED_LIBRARY" OR "${type}" STREQUAL "UNKNOWN_LIBRARY")
```
Perhaps there's an advantage to the copy-paste approach that I'm missing?